spring-cloud-starter-dataflow-server-kubernetes/pom.xml

```diff
@@ -21,5 +21,51 @@
 			<groupId>org.springframework.cloud</groupId>
 			<artifactId>spring-cloud-dataflow-server-core</artifactId>
 		</dependency>
+		<dependency>
```

Comment:
Also, I've tested this with [this initializr change](https://github.com/ericbottard/initializr/commit/947d7b32be073b6791f936e2a53dc72875a2a213) (for which I'll be happy to issue a PR once RC1 is released if you want) and the generated project emits a wrong banner.
Need to add a `description` to this POM (as this is where the token replacement happens now):

```
  ____                              ____ _                __
 / ___| _ __  _ __(_)_ __   __ _   / ___| | ___  _   _  __| |
 \___ \| '_ \| '__| | '_ \ / _` | | |   | |/ _ \| | | |/ _` |
  ___) | |_) | |  | | | | | (_| | | |___| | (_) | |_| | (_| |
 |____/| .__/|_|  |_|_| |_|\__, |  \____|_|\___/ \__,_|\__,_|
  ____ |_|    _          __|___/                 __________
 |  _ \  __ _| |_ __ _  |  ___| | _____      __  \ \ \ \ \ \
 | | | |/ _` | __/ _` | | |_  | |/ _ \ \ /\ / /   \ \ \ \ \ \
 | |_| | (_| | || (_| | |  _| | | (_) \ V  V /    / / / / / /
 |____/ \__,_|\__\__,_| |_|   |_|\___/ \_/\_/    /_/_/_/_/_/

 Spring Cloud parent pom, managing plugins and dependencies for Spring Cloud projects 1.1.0.BUILD-SNAPSHOT

```


(I can add this on merge)